lib/puppet/indirector/resource/ral.rb

```diff
@@ -12,7 +12,7 @@ def allow_remote_requests?
   def find( request )
     # find by name
     res   = type(request).instances.find { |o| o.name == resource_name(request) }
-    res ||= type(request).new(:name => resource_name(request), :audit => type(request).properties.collect { |s| s.name })
+    res ||= type(request).new(:name => resource_name(request))
```

Comment:
Jk, this is broken.

`puppet resource file /tmp/foo.txt` used to return all properties, now it only returns `ensure`.